src/Testing/TestingKit.php

```diff
@@ -0,0 +1,254 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Testing;
+
+use Nette\DI\Container;
+use PhpParser\PrettyPrinter\Standard;
+use PHPStan\Analyser\Analyser;
+use PHPStan\Analyser\NodeScopeResolver;
+use PHPStan\Analyser\TypeSpecifier;
+use PHPStan\Broker\Broker;
+use PHPStan\Broker\BrokerFactory;
+use PHPStan\Cache\Cache;
+use PHPStan\Cache\MemoryCacheStorage;
+use PHPStan\File\FileHelper;
+use PHPStan\Parser\FunctionCallStatementFinder;
+use PHPStan\Parser\Parser;
+use PHPStan\PhpDoc\PhpDocStringResolver;
+use PHPStan\PhpDoc\TypeStringResolver;
+use PHPStan\Reflection\Annotations\AnnotationsMethodsClassReflectionExtension;
+use PHPStan\Reflection\Annotations\AnnotationsPropertiesClassReflectionExtension;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Reflection\FunctionReflectionFactory;
+use PHPStan\Reflection\Php\PhpClassReflectionExtension;
+use PHPStan\Reflection\Php\PhpFunctionReflection;
+use PHPStan\Reflection\Php\PhpMethodReflection;
+use PHPStan\Reflection\Php\PhpMethodReflectionFactory;
+use PHPStan\Reflection\Php\UniversalObjectCratesClassReflectionExtension;
+use PHPStan\Reflection\PhpDefect\PhpDefectClassReflectionExtension;
+use PHPStan\Reflection\SignatureMap\SignatureMapProvider;
+use PHPStan\Rules\Registry;
+use PHPStan\Rules\Rule;
+use PHPStan\Type\FileTypeMapper;
+use PHPStan\Type\Type;
+
+final class TestingKit
+{
+
+	/** @var Container  */
+	private $container;
+
+	public function __construct(Container $container)
```

Comment:
Please don't ask for `Container` here, list every dependency directly in the constructor, then register the TestingKit in DIC.